suite-common/wallet-graph/src/graphThunks.ts

```diff
@@ -5,146 +5,333 @@ import { getBlockbookSafeTime } from '@suite-common/suite-utils';
 import TrezorConnect from '@trezor/connect';
```

Comment:
This whole file become pretty complicated, we need to write tests for this otherwise every factoring of this will be huge pain in the ass. It will probably take some extra time but I do not want to merge it without tests.

You can mock connect responses easily so you should be able to test everything.